### PR TITLE
refactor flight seq (under PBS-282)

### PR DESCRIPTION
**Pull request by bmorris-revacomm**
- refactored flight seq view (kept old views for now, just in case)
- still needs minor fixes (column rows don't quite match up, need to fix cell background colors). Just wanted to get a PR out and take a break from this and work on aircrew list for a while

### CODE REVIEW

File: PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqGrid.swift [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    @State var fromICAO         = ""
+    @State var toICAO           = ""
+    @State var takeOffTime      = ""
+    @State var landTime         = ""
+    @State var totalTime        = ""
+    @State var touchAndGo       = ""
+    @State var fullStop         = ""
+    @State var totalLandings    = ""
+    @State var sorties          = ""
+    @State var specialUse       = ""
```
Comment (Pete-Hoch) [inline PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqGrid.swift:144]: Fairly sure we do not need these anymore.

File: PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqGrid.swift [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                Text(seqLetter)
+                    .font(.pblBold(size: 18))
+                    .foregroundColor(.pblSecondary)
+                FlightRow(placeholder: "\(flight.missionNumber)", text: $missionNumber)
+                FlightRow(placeholder: "\(flight.missionSymbol)", text: $missionSymbol)
+                FlightRow(placeholder: "\(flight.fromICAO)", text: $fromICAO)
+                FlightRow(placeholder: "\(flight.toICAO)", text: $toICAO)
+                FlightRow(placeholder: "\(flight.takeOffTime)", text: $takeOffTime)
+                FlightRow(placeholder: "\(flight.landTime)", text: $landTime)
+            }
```
Comment (Pete-Hoch) [inline PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqGrid.swift:158]: I believe you want the placeholder to be some constant value and the text to be the flight.whatever thing. Look at how we are doing this in MissionDataSection.swift.

File: PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqGrid.swift [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+//                case 1: letter = "B"
+//                case 2: letter = "C"
+//                case 3: letter = "D"
+//                case 4: letter = "E"
+//                case 5: letter = "F"
+//                default: letter = ""
+//                }
+//
+//                FlightSeqColumn(flight: flight, seqLetter: letter)
+//            }
```
Comment (Pete-Hoch) [inline PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqGrid.swift:66]: We will need to add this back when we have the new data breakdown.

File: PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqView.swift [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    
+    @ObservedObject var form: Form781
+    
+    var body: some View {
+        VStack {
+            HStack {
+                Text("Flight SEQ")
+                Spacer()
+                Button {
+
```
Comment (Pete-Hoch) [inline PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqView.swift:20]: Maybe print out "Add Flight/Sorty"?

**Review by Pete-Hoch — CHANGES_REQUESTED** [review]
This is a really good start. I like the scrolling action.

File: PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqGrid.swift (hunk 3 above)
Comment (bmorris-revacomm) [reply PBLogging/NewNavViews/Form781View/FlightsSection/FlightSeqGrid.swift:66]: I couldn't get it to work correctly, that's why it's commented out :(

**Review by Pete-Hoch — APPROVED** [review]
Looks good.

**Pull request merged**